lib/migration.pm

```diff
@@ -148,7 +148,7 @@ sub check_rollback_system {
     assert_script_run("zypper mr -d -m cd -m dvd");
     # Verify registration status matches current system version
     # system is un-registered during media based upgrade
-    assert_script_run('curl -s ' . data_url('console/check_registration_status.py') . ' | python') unless get_var('MEDIA_UPGRADE');
+    assert_script_run('curl -s ' . data_url('console/check_registration_status.py') . ' | python3') unless get_var('MEDIA_UPGRADE');
```

Comment:
Met a issue today, please see: https://openqa.suse.de/tests/2274134#step/snapper_rollback/43
JSON object got type error if using python3 and I had try it by using python2 without error. Could you take a look on this?
Also I thought python3 interpreter is not always default installed on some images like minimal installation, and we only perform it on rollbacked system which always use python2 as default interpreter so far. so I suggest to still use `python` then just try to make this snippet more compatible.